src/components/ValidationMessage/CorrectIcon.tsx

```diff
@@ -0,0 +1,14 @@
+type Props = {}
+
+const CorrectIcon = ({}: Props) => {
```

Comment:
```suggestion
const CorrectIcon = () => {
```

If the component has no props, there is no need to declare them